hyperspy/signal.py

```diff
@@ -2891,6 +2861,57 @@ def var(self, axis=None, out=None):
                                                             out=out)
     var.__doc__ %= (MANY_AXIS_PARAMETER, OUT_ARG)
 
+    def nansum(self, axis=None, out=None):
```

Comment:
All `nan*` methods are new, so they require documentation in the User Guide and the header.